### PR TITLE
Preparation for a 3.4.0-beta-1 release

### PR DESCRIPTION
**Pull request by rprouse**
@jasonwilliams200OK if you have time, can you give this a review? I would like to do a beta release based on this. It includes all the recent PRs we have worked on and most importantly the port to the RTM of .NET Core.

**Do not merge** I will merge this branch back into master after the final release.

### CODE REVIEW

File: src/dotnet-test-nunit/project.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "title": "NUnit .NET Core Runner",
   "description": "NUnit console and Visual Studio runner for .NET Core and .NET 4.5.1+",
   "name": "dotnet-test-nunit",
-  "version": "3.3.0",
+  "version": "3.4.0",
```
Comment (jasonwilliams200OK) [inline src/dotnet-test-nunit/project.json]: Maybe we can add "dotnet core" in tags array and erase "dnx": https://github.com/nunit/dotnet-test-nunit/blob/31022ae3eb06dedddc4ab973f8ea9e84c9cbc3e0/src/dotnet-test-nunit/project.json#L15 :)

**Comment by jasonwilliams200OK** [comment]
LGTM.

Aside, can you please apply this minor patch to readme:


```diff
diff --git a/README.md b/README.md
index 6386801..b6179a4 100644
--- a/README.md
+++ b/README.md
@@ -48,7 +48,7 @@ I have added `"testRunner": "nunit"` to specify NUnit 3 as the test adapter. I a
 
 You can now run your tests using the Visual Studio Test Explorer, or by running `dotnet test` from the command line.
 
-```
+```sh
 # Restore the NuGet packages
 dotnet restore
 
@@ -56,7 +56,7 @@ dotnet restore
 dotnet test
 
 # Run the unit tests in a different directory
-dotnet test .\test\NUnitWithDotNetCoreRC2.Test\
+dotnet test test/NUnitWithDotNetCoreRC2.Test
```

Makes it look more cross-platform'ish (and forward slashes in path work with PowerShell too). :D

File: src/dotnet-test-nunit/project.json (hunk 1 above)
Comment (rprouse) [inline src/dotnet-test-nunit/project.json]: :+1: done

**Comment by rprouse** [comment]
@jasonwilliams200OK thanks for the feedback. I have implemented both of your suggestions. The beta should be out tonight.

**Comment by rprouse** [comment]
Travis on Linux passed, Mac is still hanging so I cancelled them.

**Comment by jasonwilliams200OK** [comment]
Travis status bot is suggesting that they have started to process the backlog. Its a matter of time our "ticket lock" gets unlocked :smile: 
https://twitter.com/traviscistatus

**Comment by jasonwilliams200OK** [comment]
@rprouse, fixed the travis script with minor improvements. Also updated Readme example after testing on macOS and Windows. i.e. we just need one item in `imports` and NUnit deps besides dotnet-test-nunit.

File: .travis.yml [hunk 2]
```diff
@@ -30,7 +30,6 @@ matrix:
 before_install:
   # Install OpenSSL
   - if test "$TRAVIS_OS_NAME" == "osx"; then
-      brew update;
```
Comment (jasonwilliams200OK) [inline .travis.yml]: This is just to save a couple of seconds per job (coz we don't care!). :wink:

**Pull request closed without merging**